fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -302,6 +303,8 @@ protected void configure() {
 
                 bind(getClock()).to(Clock.class);
 
+                bind(getHttpResponseMaker()).to(HttpResponseMaker.class);
```

Comment:
I disagree. This is a dirt-simple binding. All we'd be testing is the HK2 injection framework. Far as I can tell we test hardly any of the other similarly simple bindings (i.e. we don't verify that the `ObjectMapperSuite` is being bound correctly.